rpcs3/Emu/Cell/lv2/sys_prx.cpp

```diff
@@ -25,6 +25,7 @@ LOG_CHANNEL(sys_prx);
 // <string: firmware sprx, int: should hle if 1>
 extern const std::map<std::string_view, int> g_prx_list
 {
+	{ "/dev_flash/sys/internal/libfs_utility_init.sprx", 1 },
```

Comment:
Edit comment in line 229 because now it is used, fun fact when I refactored sys_prx code I added the / case just for this code.